test/e2e/commands/network.test.mjs

```diff
@@ -0,0 +1,114 @@
+  describe,
+  expect,
+  it
+} from '@jest/globals'
+import {
+  bootstrapTestVariables,
+  getDefaultArgv
+} from '../../test_util.js'
+import {
+  constants
+} from '../../../src/core/index.mjs'
+import { flags } from '../../../src/commands/index.mjs'
+import * as version from '../../../version.mjs'
+
+describe('NetworkCommand', () => {
+  const testName = 'network-cmd-e2e'
+  const namespace = testName
+  const argv = getDefaultArgv()
+  argv[flags.namespace.name] = namespace
+  argv[flags.releaseTag.name] = 'v0.47.0-alpha.0'
+  argv[flags.keyFormat.name] = constants.KEY_FORMAT_PEM
+  argv[flags.nodeIDs.name] = 'node0,node1,node2'
+  argv[flags.generateGossipKeys.name] = true
+  argv[flags.generateTlsKeys.name] = true
+  argv[flags.deployMinio.name] = true
+  argv[flags.fstChartVersion.name] = version.FST_CHART_VERSION
+  argv[flags.force.name] = true
+
+  const bootstrapResp = bootstrapTestVariables(testName, argv)
+  const k8 = bootstrapResp.opts.k8
+  const accountManager = bootstrapResp.opts.accountManager
+  const configManager = bootstrapResp.opts.configManager
+
+  const networkCmd = bootstrapResp.cmd.networkCmd
+  const clusterCmd = bootstrapResp.cmd.clusterCmd
+
+  afterAll(async () => {
+    await k8.deleteNamespace(namespace)
+    await accountManager.close()
+  })
+
+  beforeAll(async () => {
+    await clusterCmd.setup(argv)
+  })
+
+  it('network deploy command should succeed', async () => {
+    expect.assertions(4)
+    try {
+      await expect(networkCmd.deploy(argv)).resolves.toBeTruthy()
+
+      // check pod names should match expected values
+      await expect(k8.getPodByName('network-node0-0'))
+        .resolves.toHaveProperty('metadata.name', 'network-node0-0')
+      await expect(k8.getPodByName('network-node1-0'))
+        .resolves.toHaveProperty('metadata.name', 'network-node1-0')
+      await expect(k8.getPodByName('network-node2-0'))
+        .resolves.toHaveProperty('metadata.name', 'network-node2-0')
+      // get list of pvc using k8 listPvcsByNamespace function and print to log
+      const pvcs = await k8.listPvcsByNamespace(namespace)
+      networkCmd.logger.showList('PVCs', pvcs)
+    } catch (e) {
+      networkCmd.logger.showUserError(e)
+      expect(e).toBeNull()
+    }
+  }, 60000)
+
+  it('network destroy should success', async () => {
+    argv[flags.deletePvcs.name] = true
+    configManager.update(argv, true)
+
+    expect.assertions(3)
+    try {
+      await expect(networkCmd.destroy(argv)).resolves.toBeTruthy()
+
+      while ((await k8.getPodsByLabel(['fullstack.hedera.com/type=network-node'])).length > 0) {
+        networkCmd.logger.debug('Pods are still running. Waiting...')
+        await new Promise(resolve => setTimeout(resolve, 3000))
+      }
+
+      while ((await k8.getPodsByLabel(['app=minio'])).length > 0) {
+        networkCmd.logger.showUser('Waiting for minio container to be deleted...')
+        await new Promise((resolve) => setTimeout(resolve, 3000))
```

Comment:
You can use a helper function we have here:https://github.com/hashgraph/solo/blob/48c5b77ebe05e6301443dc76ac5cf8e94dfc0b33/src/core/helpers.mjs#L30
```
import {sleep } from '../core/helpers.mjs'
....
await sleep(2000)
```

Reply:
Updated. thanks